E-Commerce/readme.md

```diff
@@ -45,13 +45,81 @@ erDiagram
 
 Write SQL queries to complete the following tasks:
 
-- [ ] List all the products whose name contains the word "socks"
-- [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
-- [ ] List the 5 most expensive products
+- [x] List all the products whose name contains the word "socks"
+
+```psql
+SELECT * FROM products
+WHERE product_name LIKE '%socks%'
+```
+
+- [x] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+
+```psql
+SELECT p1.prod_id, p1.supp_id, p1.unit_price, p2.product_name FROM product_availability p1
+JOIN products p2
+ON p1.prod_id = p2.id
+WHERE unit_price >= 100
+```
+
+- [x] List the 5 most expensive products
+
+```psql
+SELECT * FROM product_availability
+WHERE unit_price >= 10
+ORDER BY unit_price DESC
+LIMIT 5
+```
+
 - [ ] List all the products sold by suppliers based in the United Kingdom. The result should only contain the columns product_name and supplier_name
-- [ ] List all orders, including order items, from customer named Hope Crosby
-- [ ] List all the products in the order ORD006. The result should only contain the columns product_name, unit_price, and quantity
-- [ ] List all the products with their supplier for all orders of all customers. The result should only contain the columns name (from customer), order_reference, order_date, product_name, supplier_name, and quantity
+
+```psql
+SELECT s.supplier_name, p.product_name
+FROM suppliers s
+JOIN  products p
```

Comment:
After the JOIN command, you'll need the ON command to tell the database which columns to use to join the two tables together.

The ON command you have in the next line is used for filtering, It should be changed to a WHERE clause.

Reply:
solve it